src/TCPClient.h

```diff
@@ -65,6 +65,13 @@ class TCPClient : public Socket {
      * Open a new connection to the remote host.
      */
     void connect();
+    
+    /**
+     * Close the existing connection to the remote host.
+     * Added on 04/03/2017 by Alex.CUER in order to post datas in an emoncms server
```

Comment:
There is no need to add change logs to the source code - this is maintained in git instead.